lib/ansible/modules/packaging/os/package.py

```diff
@@ -33,7 +33,7 @@
   state:
     description:
       - Whether to install (C(present), or remove (C(absent)) a package. Other states depend on the underlying package module, i.e C(latest).
-    required: true
+    required: false
```

Comment:
package is a special case, note this is a "wrapper" of the underlying package manager. Some may not have a default defined for state (present). requried=true is a safe fallback.

Reply:
its required as that is the 'lowest common denominator' for package modules, otherwise it will 'fail sometimes'.